webnode/config/webpack.config.dev.js

```diff
@@ -222,6 +222,10 @@ module.exports = {
         "eth-address": "0xD1833A50f411432aD38E8374df8Cfff79e743788"
       }
     }),
+    new webpack.EnvironmentPlugin({
+      NODE_ENV: "development",
+      DEBUG: true
```

Comment:
Does this need to be a separate variable?

Reply:
thought the same thing, decided to keep it so we can stay in development mode but toggle certain things